src/dosbox.cpp

```diff
@@ -377,9 +377,15 @@ static void DOSBOX_RealInit(Section * sec) {
 	} else if (mtype == "svga_paradise") {
 		svgaCard = SVGA_ParadisePVGA1A;
 	} else if (mtype == "vgaonly") {
-		svgaCard = SVGA_None;
-	} else
+		svgaCard = SVGA_ParadisePVGA1A;
+		LOG_WARNING("CONFIG: Deprecated option 'machine = vgaonly', now set to 'svga_paradise'");
```

Comment:
Great that you're logging a deprecation warning 💯 
Would rephrasing the message like this make things a bit clearer perhaps?

```
CONFIG: 'machine = vgaonly' is deprecated and now maps to 'svga_paradise'; please use 'machine = svga_paradise' instead
```

Reply:
Added with slight change to recommend defaults:

```
LOG_WARNING("CONFIG: 'machine = vgaonly' is deprecated. Consider using "
"defaults or 'machine = svga_paradise' if basic (S)VGA is critical.");
```